src/groups/mqb/mqbs/mqbs_filebackedstorage.cpp

```diff
@@ -254,6 +256,8 @@ void FileBackedStorage::setQueue(mqbi::Queue* queue)
 
     // Update queue stats if a queue has been associated with the storage.
     if (queue) {
+        queue->setStats(d_queueStats_sp);
+
```

Comment:
Double-checking the logic.

1) Suppose, we have storage with 10 messages, no queue.  Broker starts, reads the storage, calls `FileBackedStorage::processMessageRecord`.
 `FileBackedStorage::numMessages` would return `10`.  The stats must have `10`

2) Then, 5 more messages get replicated, `FileBackedStorage::processMessageRecord` updates the stats; now `numMessages()` return `15`, the stats must have `15`

3) Then queue gets created and `FileBackedStorage::setQueue` sets the stats as `15`.

Seems, like `setQueueContentRaw()` is redundant now?  Otherwise, we have incorrect stats at some point?

Reply:
Yes, that's right. Removed `setQueueContentRaw()` call